.gitignore

```diff
@@ -21,3 +21,114 @@ bin/
+athena.exceptions/index_SORT_BY_CLASS.html
+athena.exceptions/index_SORT_BY_CLASS_DESC.html
+athena.exceptions/index_SORT_BY_LINE.html
+athena.exceptions/index_SORT_BY_LINE_DESC.html
+athena.exceptions/index_SORT_BY_METHOD.html
+athena.exceptions/index_SORT_BY_METHOD_DESC.html
+athena.exceptions/index_SORT_BY_NAME_DESC.html
+athena.exceptions/.classes/AddMissingRequiredParametersException.html
+athena.exceptions/.classes/CommandException.html
+athena.exceptions/.classes/DeleteInvalidIndexException.html
+athena.exceptions/.classes/DeleteNoIndexException.html
+athena.exceptions/.classes/DoneInvalidIndexException.html
+athena.exceptions/.classes/DoneNoIndexException.html
+athena.exceptions/.classes/EditNoIndexException.html
+athena.exceptions/.classes/EmptyTaskListException.html
+athena.exceptions/.classes/InvalidCommandException.html
+athena.exceptions/.classes/TaskNotFoundException.html
+athena.exceptions/.classes/ViewInvalidIndexException.html
+athena.exceptions/.classes/ViewNoIndexException.html
+athena.task/index.html
+athena.task/index_SORT_BY_BLOCK.html
+athena.task/index_SORT_BY_BLOCK_DESC.html
+athena.task/index_SORT_BY_CLASS.html
+athena.task/index_SORT_BY_CLASS_DESC.html
+athena.task/index_SORT_BY_LINE.html
+athena.task/index_SORT_BY_LINE_DESC.html
+athena.task/index_SORT_BY_METHOD.html
+athena.task/index_SORT_BY_METHOD_DESC.html
+athena.task/index_SORT_BY_NAME_DESC.html
+athena.task.taskfilter/index.html
+athena.task.taskfilter/index_SORT_BY_BLOCK.html
+athena.task.taskfilter/index_SORT_BY_BLOCK_DESC.html
+athena.task.taskfilter/index_SORT_BY_CLASS.html
+athena.task.taskfilter/index_SORT_BY_CLASS_DESC.html
+athena.task.taskfilter/index_SORT_BY_LINE.html
+athena.task.taskfilter/index_SORT_BY_LINE_DESC.html
+athena.task.taskfilter/index_SORT_BY_METHOD.html
+athena.task.taskfilter/index_SORT_BY_METHOD_DESC.html
+athena.task.taskfilter/index_SORT_BY_NAME_DESC.html
+athena.task.taskfilter/.classes/FlexibleTimeFilter.html
+athena.task.taskfilter/.classes/ForecastFilter.html
+athena.task.taskfilter/.classes/ImportanceFilter.html
+athena.task.taskfilter/.classes/TaskFilter.html
+athena.task/.classes/Task.html
+athena.task/.classes/Time.html
+athena.timetable/index.html
+athena.timetable/index_SORT_BY_BLOCK.html
+athena.timetable/index_SORT_BY_BLOCK_DESC.html
+athena.timetable/index_SORT_BY_CLASS.html
+athena.timetable/index_SORT_BY_CLASS_DESC.html
+athena.timetable/index_SORT_BY_LINE.html
+athena.timetable/index_SORT_BY_LINE_DESC.html
+athena.timetable/index_SORT_BY_METHOD.html
+athena.timetable/index_SORT_BY_METHOD_DESC.html
+athena.timetable/index_SORT_BY_NAME_DESC.html
+athena.timetable/.classes/Timetable.html
+athena.timetable/.classes/TimetableDay.html
+athena/.classes/Athena.html
+athena/.classes/Forecast.html
+athena/.classes/Importance.html
+athena/.classes/Parser.html
+athena/.classes/Recurrence.html
+athena/.classes/Storage.html
+athena/.classes/TaskList.html
+athena/.classes/TimeAllocator.html
+athena/.classes/Ui.html
```

Comment:
What are these?

Reply:
Its the code coverage report files, they might appear again so I put it here so it won't be added accidentally